app/utils/response_router.js

```diff
@@ -234,7 +234,27 @@ const actions = {
   }
 }
 
-exports.response = (req, res, actionName, options) => {
+exports.errorResponse = function errorReponse (req, res, reason, options, error) {
+  logger.error('Rendering error response', {
+    page: 'ERROR',
+    reason: reason,
```

Comment:
Very minor but these can just be
```
{
  reason,
  error
}
```

when the key/ value pairs match

Reply:
👍 done